test/benchmarks/src/main/scala/scala/reflect/ClassTagBenchmark.scala

```diff
@@ -0,0 +1,30 @@
+package scala.reflect
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+import org.openjdk.jmh.infra.Blackhole
+
+
+@BenchmarkMode(Array(Mode.AverageTime))
+@Fork(2)
+@Threads(1)
+@Warmup(iterations = 10)
+@Measurement(iterations = 10)
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+@State(Scope.Benchmark)
+class ClassTagBenchmark {
+  @OperationsPerInvocation(10000000)
+  @Benchmark def doubleClassTagUnapply(bh: Blackhole): Any = {
+    val ct = classTag[Double]
+    for (1 <- 1 to 10000000) {
+      bh.consume(ct.unapply(1.0))
+    }
```

Comment:
It is a bit of code-smell to loop inside of a JMH `@Benchmark` method. I assume you're doing that to amortize the cost of line 19. But it would be better to move that code to a `@Setup` method instead, and assign the classtag to a field for use here.